Automattic/jetpack
# Boost: Fix overlay overlapping issues and update hard-coded post types

Fixes https://github.com/Automattic/boost-cloud/issues/296 and https://github.com/Automattic/boost-cloud/issues/302

## Proposed changes:

* Fix overlays showing under the next section;
* Update list of post types for "Other" group in ISA UI, to be fetched from the current website.
<img width="327" alt="image" src="https://github.com/Automattic/jetpack/assets/11799079/901f940b-8837-492c-b068-13be9b90a854">


### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup this version of Boost;
* Make sure overlays (Score explanation and "Other" group from ISA UI) are showing properly;
* Make sure that the custom post types listed in the "Other" group tooltip are the correct ones.
